packages/node_modules/@node-red/registry/lib/installer.js

```diff
@@ -235,12 +235,17 @@ function checkPrereq() {
         return Promise.resolve();
     } else {
         return new Promise(resolve => {
-            child_process.execFile(npmCommand,['-v'],function(err) {
+            child_process.execFile(npmCommand,['-v'],function(err,stdout) {
                 if (err) {
                     log.info(log._("server.palette-editor.npm-not-found"));
                     paletteEditorEnabled = false;
                 } else {
-                    paletteEditorEnabled = true;
+                    if (parseInt(stdout.split(".")[0]) < 3) {
```

Comment:
Isn't npm 1.x the main problem? This will drop support for npm 2 which actually works quite well.

Reply:
Well node 6 comes with npm 3 so thought I would match up with eth version of node we (will) support